src-ui/lib/drizzle/schema.ts

```diff
@@ -1,19 +1,21 @@
-import { sqliteTable, AnySQLiteColumn, text, integer } from "drizzle-orm/sqlite-core"
-  import { sql } from "drizzle-orm"
-
-export const books = sqliteTable("books", {
-	id: text().default("sql`(uuid_blob(uuid()))`").primaryKey().notNull(),
-	isbn10: text(),
-	isbn13: text(),
-	title: text().notNull(),
-	subtitle: text(),
-	authors: text().notNull(),
-	tags: text(),
-	series: text(),
-	pageCount: integer(),
-	publicationDate: text(),
-	copyrightDate: text(),
-	coverImages: text(),
-	hasRead: integer(),
-});
+import { sqliteTable, text, integer } from 'drizzle-orm/sqlite-core'
 
+export const books = sqliteTable('books', {
+  id: text().default('sql`(uuid_blob(uuid()))`').primaryKey().notNull(),
+  isbn10: text(),
+  isbn13: text(),
+  title: text().notNull(),
+  subtitle: text(),
+  // authors: text().notNull(), // NOTE: manually keep the line below after schema pull overrides it.
+  authors: text({ mode: 'json' }).notNull().$type<string[]>(),
```

Comment:
NOTE: the commented out lines are what gets auto-generated by the `gen:schema` script; and the more complex ones are hand-written: `{mode: 'json'}` and `$type<string[]>()`.

**whenever** we re-generate the schema (after a migration) we will need to make sure these changes are not discarded.